commands.go

```diff
@@ -1094,8 +1094,14 @@ type ZRangeByScore struct {
 	Offset, Count int64
 }
 
-func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores bool) *StringSliceCmd {
-	args := []interface{}{"ZRANGEBYSCORE", key, opt.Min, opt.Max}
+func (c *commandable) zRangeByScore(key string, opt ZRangeByScore, withScores, isLex bool) *StringSliceCmd {
```

Comment:
+1, please refactor it so following is possible
- zRange("ZRANGEBYLEX", key, opts, withScores)
- zRange("ZRANGEBYSCORE", key, opts, withScores)